packages/vuetify/src/components/VForm/__tests__/VForm.spec.ts

```diff
@@ -1,27 +1,58 @@
+// Libraries
 import Vue from 'vue'
+import Vuetify from '../../../framework'
+
+// Components
+import VForm from '../VForm'
```

Comment:
Is there anything of value added by testing all of the input components? All validation should be handled in the validatable mixin.